dotnet/aspnetcore22/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using System;
+
+namespace SampleWebApplication.FunctionalTests
+{
+    [TestClass]
+    public class SampleFunctionalTests
+    {
+        private static TestContext testContext;
+        private RemoteWebDriver driver;
+
+        [ClassInitialize]
+        public static void Initialize(TestContext testContext)
+        {
+            SampleFunctionalTests.testContext = testContext;
+        }
+
+        [TestMethod]
+        public void SampleFunctionalTest1()
+        {
+            try
+            {
+                driver = GetChromeDriver();
+                var webAppUrl = testContext.Properties["webAppUrl"].ToString();
+                driver.Navigate().GoToUrl(webAppUrl);
```

Comment:
We have added retry in these methods since sometimes web app takes time to initialize. Can you follow the newer pattern to add retry

Reply:
Sorry - I don't understand … I used what was already there and it worked fine.
I no longer have the infra setup to retest (as discussed with Pulkit)

>> From: Pulkit Agarwal
>> For the infrastructure, just take a screenshot of website of all the updated section and feel free to delete the infra.

Reply:
Ok - let me look at this

Reply:
I've made the changes - and tested that they work